edge-modules/snitch/src/settings.yaml

```diff
@@ -1,9 +1,19 @@
+build_id: 239492391
 test_duration:
   secs: 28800
   nanos: 0
-alert_url: http://localhost/
-influx_url: http://localhost/
+alert:
+  host: http://localhost/
+  path: /workflows/id/triggers/manual/paths/invoke
+  query:
+    api-version: 2016-10-01
+    sp: /triggers/manual/run
+    sv: 1.0
+    sig: password
+influx_url: http://influxdb/
```

Comment:
default influx url should be http://influxdb:8086

Reply:
Done